src/App.jsx

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import AnimeListView from './components/AnimeListView';
+import AnimeView from './components/AnimeView';
+import SearchBar from './components/SearchBar';
 
 const App = () => (
   <div>
-    App
+    <SearchBar />
+    <Routes>
+      <Route path="/" element={<AnimeListView />} />
+      <Route path="/anime/:id" element={<AnimeView />} />
+    </Routes>
   </div>
 );
```

Comment:
- [ ] Your App looks great. However, per the project requirements, you need to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)) including:
   * Typography: font face, size, and weight
   * Layout: composition and space between elements.
- [ ] However, in the mobile view of your app, the homepage is not following the layout as seen in the Behance design. 
- You can see that the Behance design has the contents of the page split in two and the boxes containing each country don't have spaces between. Aside from this, each box has an arrow that when clicked on takes a user to the details page but this wasn't incorporated in your design so kindly include them as well :wink: 
- I also noticed that you are missing the `navbar` as seen in the Behance design so kindly include that as well :wink: 

| Your Design | The Behance Design |
|--------|--------|
| ![image](https://github.com/10-menachi/anime-facts/assets/102699958/1c0efa6d-be11-4184-b215-7dbb38c07576) |![image](https://github.com/10-menachi/anime-facts/assets/102699958/44473db8-abce-45ee-b17f-2a70d036daf3) |